Python 3.4.1 (v3.4.1:c0e311e010fc, May 18 2014, 10:38:22) [MSC v.1600 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> while (a<50):
	print ("No Retakes")
	a += 1

	
Traceback (most recent call last):
  File "<pyshell#3>", line 1, in <module>
    while (a<50):
NameError: name 'a' is not defined
>>> while (a<50):
	print ("No Retakes")
	a += 1

Traceback (most recent call last):
  File "<pyshell#5>", line 1, in <module>
    while (a<50):
NameError: name 'a' is not defined
>>> 
>>> 
>>> a=0
>>> while (a<50):
	print ("No Retakes")
	a += 1

No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
No Retakes
>>> ================================ RESTART ================================
>>> 
win
3
>>> print ("*"*5)
*****
>>> ================================ RESTART ================================
>>> 
55
5
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
>>> ================================ RESTART ================================
>>> 
7
7
>>> 
>>> 
>>> ================================ RESTART ================================
>>> 
10

Traceback (most recent call last):
  File "C:/Python34/55.py", line 5, in <module>
    m= int (input())
ValueError: invalid literal for int() with base 10: ''
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/55.py", line 4, in <module>
    m= int (input())
ValueError: invalid literal for int() with base 10: ''
>>> 10
10
>>> 6
6
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/55.py", line 4, in <module>
    m = int (input())
ValueError: invalid literal for int() with base 10: ''
>>> 
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/23.py", line 4, in <module>
    m = int (input())
ValueError: invalid literal for int() with base 10: ''
>>> 12
12
>>> 
>>> 4242
4242
>>> ================================ RESTART ================================
>>> 
3
up

>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/98.py", line 1, in <module>
    n=int (input())
ValueError: invalid literal for int() with base 10: ''
>>> 54
54
>>> ================================ RESTART ================================
>>> 

>>> ================================ RESTART ================================
>>> 
65
5
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
*****
>>> 
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/98.py", line 1, in <module>
    n=int (input())
ValueError: invalid literal for int() with base 10: ''
>>> ================================ RESTART ================================
>>> 
2
2
*****
*****
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/98.py", line 1, in <module>
    n=int (input())
ValueError: invalid literal for int() with base 10: ''
>>> 2
2
>>> 
>>> ================================ RESTART ================================
>>> 

Traceback (most recent call last):
  File "C:/Python34/98.py", line 1, in <module>
    n=int (input())
ValueError: invalid literal for int() with base 10: ''
>>> ================================ RESTART ================================
>>> 
2
2
**
**
>>> 
